rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -203,6 +278,28 @@ void save_manager_dialog::UpdateList()
 	resize(preferredSize.boundedTo(maxSize));
 }
 
+void save_manager_dialog::UpdateIcons()
+{
+	const int dpr = devicePixelRatio();
+	const int width = m_icon_size.width() * dpr;
+	const int height = m_icon_size.height() *dpr * 176 / 320;
```

Comment:
I don't wanna be an ass but this is missing a space :(

Reply:
Sigh.